packages/phone-number-privacy/signer/src/common/key-management/mock-key-provider.ts

```diff
@@ -5,13 +5,22 @@ import {
 import { DefaultKeyName, Key, KeyProviderBase } from './key-provider-base'
 
 export class MockKeyProvider extends KeyProviderBase {
+  private pnpKey: string
+  private domainsKey: string
+
+  constructor(pnpKey?: string, domainsKey?: string) {
```

Comment:
```suggestion
  constructor(private pnpKey: string = PNP_DEV_SIGNER_PRIVATE_KEY, private domainsKey: string = DOMAINS_DEV_SIGNER_PRIVATE_KEY) {
```
nit: ts has some cool syntax for this https://www.typescripttutorial.net/typescript-tutorial/typescript-default-parameters/#:~:text=Optional%20parameters%20must%20come%20after,get%20the%20default%20initialized%20value.